sw/device/silicon_creator/rom_ext/BUILD

```diff
@@ -272,6 +272,16 @@ manifest(d = {
     "security_version": ROM_EXT_VERSION.SECURITY,
 })
 
+manifest(d = {
+    "name": "manifest_perso",
```

Comment:
maybe we should move this to the `sw/device/silicon_creator/manuf/base/BUILD` file? so it is next to where it is used?

Reply:
Okay, done.